### ISSUE TITLE
Ys NES Definitive Edition

### ISSUE THREAD

**Issue by hossbags2** [TASEmulators/fceux#713]
**Describe the bug**
The game starts up, title screen is ok. Once you start he game, the graphics are scrambled

**Expected behavior**
the game should run without garbled graphics

**Screenshots**

**(please complete the following information):**
Windows 10
latest build  - 1.0.1827

**Comment by dragon2snow** [comment]
PPU BUG
switch to New PPU
![image](https://github.com/TASEmulators/fceux/assets/4350811/28146988-e0da-478d-94d2-2098d73a2e5b)

**Comment by darthvaderx** [comment]
“Hello, I am the person responsible for this hack. In fact, it is a correction of another one that had numerous bugs, making the gameplay experience completely unpleasant. After a lot of work, I managed to solve most of these problems. However, what I didn’t expect was that it wouldn’t work correctly in FCEUX. Trying to fix this compatibility problem would require me to start over from scratch, and I’m not very willing to do that. Fortunately, it still works in Mesen and Nestopia, as was warned in the text that comes with the patch. I don’t know if it works in other emulators or flashcards because I didn’t have the time or desire to test them. This is just a clarification, ok?”

**Comment by negativeExponent** [comment]
@darthvaderx did you remove the link to the patch from romhacking.net? cant find it there anymore.

**Comment by darthvaderx** [comment]
No, it's just that it was changed from the 'hack' category to 'translation addendum' along with Ys 2 and 3:

https://www.romhacking.net/translations/7230/

https://www.romhacking.net/translations/7228/

https://www.romhacking.net/translations/7229/

**Comment by negativeExponent** [comment]
ok. thanks for the links

i think this can be considered closed since the patch is playable if newppu mode is set. i've had the patch working in my local fork of fceumm as well.
![Ys NES Definitive Edition-240226-114434](https://github.com/TASEmulators/fceux/assets/54053706/790d2032-1223-4e0e-a7af-d905d0ee336c)

**Comment by darthvaderx** [comment]
And I released new versions considered final of the three where I made small touches to the title screens, considered final because I won't work on them anymore.
